File: src/tests/Markify.Fixtures/Projects/Source/Interface/InterfacePartial.cs
using System;
using System.Collections;

public partial interface FooType : IDisposable
{
    int PropertyOne { get; set; }

    event EventHandler Done;

    void MethodOne(int foo);
}

public partial interface FooType : IEnumerable
{
    int PropertyTwo { get; set; }

    event EventHandler Started;

    void MethodTwo(int foo, int bar);
}

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Interface/InterfacePartial.vb
Imports System
Imports System.Collections

Public Partial Interface FooType
    Inherits IDisposable
    Property PropertyOne As Integer

    Event Done As EventHandler

    Sub MethodOne(foo As Integer)
End Interface

Public Partial Interface FooType
    Inherits IEnumerable
    Property PropertyTwo As Integer

    Event Started As EventHandler

    Sub MethodTwo(foo As Integer, bar As Integer)
End Interface